nixos/modules/services/networking/wg-quick.nix

```
@@ -17,6 +17,13 @@ let
         description = "The IP addresses of the interface.";
       };
 
+      autostart = mkOption {
+        description = "Start the systemd-unit automatically.";
```

```suggestion
        description = "Whether to bring up this interface automatically during boot.";
```
talking about the entire interface seems more clear here since the unit is a bit of an impementation detail.